src/main/include/subsystems/Shooter.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <rev/CANSparkMax.h>
+#include <ctre/phoenix/motorcontrol/can/TalonSRX.h>
+#include <units/math.h>
+#include <frc/controller/PIDController.h>
+#include <frc/DigitalInput.h>
+
+#include "Constants.h"
+
+class Shooter
+{
+    public:
+
+    Shooter();
+
+    void runShooter(const double motor_output);
+
+    void runIntake(const double motor_output);
+
+    void moveToAngle(const double angle);
+
+    void periodic();
+
+    void init();
+
+    rev::CANSparkMax m_firing_motorL;
+    rev::CANSparkMax m_firing_motorR;
+    rev::CANSparkMax m_arm_motorT;
+    rev::CANSparkMax m_arm_motorB;
+
+    rev::SparkMaxAbsoluteEncoder m_arm_encoderT;
+    rev::SparkMaxAbsoluteEncoder m_arm_encoderB;
```

Comment:
Suggest going ahead and spelling out "left," "right," "top," and "bottom." Three reasons:

1. You already do so in Climber.h, e.g., `m_left_pulley`.
2. Strictly speaking, `m_firing_motorR`, for instance, is not proper snake_case/underscore_case. It's a bit of a mashup of snake_case and camelCase.
3. The other night when you were walking me through this code, at first I missed what you said `m_arm_motorT` and `m_arm_motorB` were. Before it finally dawned on me, I kept finding myself distracted, wondering what `T` and `B` signified. It's an example of one of those things that is obvious to you in the moment when you write it, but not necessarily obvious to someone coming to the code cold. That person could even be _you_ when you come back to the code a week later.

```suggestion
    rev::CANSparkMax m_firing_motor_left;
    rev::CANSparkMax m_firing_motor_right;
    rev::CANSparkMax m_arm_motor_top;
    rev::CANSparkMax m_arm_motor_bottom;

    rev::SparkMaxAbsoluteEncoder m_arm_encoder_top;
    rev::SparkMaxAbsoluteEncoder m_arm_encoder_bottom;
```